randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -1040,7 +1042,12 @@ Extra - room_id: [28]
   > Pickup (Energy Tank)
       All of the following:
           Speed Booster
-          Space Jump or Can Activate Pillar
+          Any of the following:
+              Space Jump or Can Activate Pillar
+              # Wall Jump and Shinespark straight up - https://youtu.be/Pvv2dYRZAj4
+              Wall Jump (Intermediate)
+              # Wall Jump and ignore the pillar - https://youtu.be/FJxCd30N2J4
+              Wall Jump (Advanced)
```

Comment:
I wonder if this should be an expert walljump instead?

Reply:
I had only set this as "advanced" because there's not really any negative consequence of failing it as you can retry pretty much instantly. The only negative consequence you would really have is spending too much time on a trick. I can still bump it up if you think expert better suits this trick.

Reply:
how precise do you have to be for the trick? cause it looked pretty precise for me.

Reply:
yeah after trying for a bit, it definitely feels like expert based on the other tricks we have in advanced.

Reply:
I suppose it is fairly precise. I'll bump it up. If anything, having the other trick be advanced means this one probably should be bumped up.